proxy/src/main/java/com/wavefront/agent/handlers/AbstractSenderTask.java

```diff
@@ -55,6 +55,7 @@
   final Counter blockedCounter;
   final Counter bufferFlushCounter;
   final Counter bufferCompletedFlushCounter;
+  private final Histogram metricSize;
```

Comment:
[future] we may want to call it atomSizes (or metricSizes in the future) but we aren't consistent about the term atom or metric so this works :)

Reply:
sure